package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "blueprint",
   "displayName": "Blueprint - New Files and Folders of Files from Templates",
-  "description":
-    "Easily create files and folder of files from templates in Visual Studio Code",
-  "version": "1.5.0",
+  "description": "Easily create files and folder of files from templates in Visual Studio Code",
+  "version": "1.6.0",
```

Comment:
Since this is a breaking change I'll make this the 2.0.0 release -- I can make that change during release though.